includes/class-magic-link.php

```diff
@@ -311,17 +311,18 @@ private static function generate_url( $user, $url = '' ) {
 	 *
 	 * @param \WP_User $user        User to generate the magic link for.
 	 * @param string   $redirect_to Which page to redirect the reader after authenticating.
+	 * @param string   $subject The subject of the email.
+	 * @param string   $message     Message to show in body of email.
 	 *
 	 * @return array|\WP_Error $email Email arguments or error. {
 	 *   Used to build wp_mail().
 	 *
 	 *   @type string $to      The intended recipient - New user email address.
-	 *   @type string $subject The subject of the email.
```

Comment:
This may have been by accident?

Reply:
Yes, I put the `@param` line in the wrong spot at first.

Reply:
This line should be restored

Reply:
Whoops, I misunderstood what you were saying—restored in ae476dd.